src/components/templates/artefact-search/artefact-search.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react';
+
+import { observer } from 'mobx-react-lite';
+
+import ArtefactOverview from '~/components/organisms/artefact-overview';
+import SearchSidebar from '~/components/organisms/search-sidebar';
+
+import './artefact-search.scss';
+
+import StoreContext from '~/store/StoreContext';
+
+const ArtefactSearch = () => {
+  const { globalSearch } = useContext(StoreContext);
+
+  return (
+    <div
+      className="artefact-search"
+      data-component="organisms/artefact-search"
+    >
+      <div className="artefact-search__results-area">
+      { globalSearch.loading && 'Loading...' }
+      { !globalSearch.loading
+        && <ArtefactOverview
+          items={ globalSearch.flattenedSearchResultItems }
```

Comment:
Ausgabe aller Suchergebnisse über die `ArtefactOverview`-Komponente.
Hierfür wird auf den Computed-Value `flattenedSearchResultItems` gesetzt, um an dieser Stelle nicht selbst die Ergebnisse zu "flatten".
Wir profitieren hierbei davon, dass der Computed-Value im Store stets aktuell gehalten wird, sollte sich an den Ergebnissen etwas ändern. Wird `flattenedSearchResultItems` neu berechnet, können Komponenten auf diese Neuberechnung reagieren. Vorteilhaft ist zudem, dass das "flatten" im Store angesiedelt ist, wodurch man das "flatten" nicht mehr selbst auf halben Wege in einer Komponente durchführen muss.